docker/prod

```diff
@@ -20,6 +20,7 @@ ENV LANGUAGE en_US.UTF-8
 
 COPY requirements /app/requirements/
 RUN pip install --no-cache-dir --require-hashes --no-deps -r requirements/prod.txt
+RUN pip install psycopg2
```

Comment:
Is `psycopg2` necessary?

Because current env looks still using MySQL.

https://github.com/mozilla/remo/blob/51b1c00998d18d19f58ee8ad0b45c47cc79ca0ca/env-dist#L27

Also, `prod.txt` file does not contain `psycopg2`.

https://github.com/mozilla/remo/blob/d7592081db3876fc335f0caa1172b8f03087af66/requirements/prod.txt#L3-L362

Reply:
Yes its used on heroku.